pkg/tasks/invalid.go

```diff
@@ -21,7 +21,11 @@ func (u *Invalid) header() string {
 	return ""
 }
 
-func (u *Invalid) perform(ctx *Context) (err error) {
+func (u *Invalid) perform(ctx *context) (err error) {
 	ctx.ui.TaskWarning(fmt.Sprintf("%s: %+v", u.err, u.definition))
 	return nil
```

Comment:
maybe it would make sense to return an error by default here since it's the invalid task